rll-examples/install-packages.sh

```diff
@@ -0,0 +1,114 @@
+#! /bin/bash
+# ---
+# RightScript Name: SYS packages install
+# Description: |
+#   Installs packages required by RightScripts.
+#   To handle naming variations, prefix package names with "yum:" or "apt:".
+# Inputs:
+#   PACKAGES_INSTALL:
+#     Input Type: single
+#     Category: System
+#     Description: |
+#       Set to "false" in order to skip combined package installation of RS_PACKAGES.
+#     Required: true
+#     Advanced: true
+#     Default: text:true
+#     Possible Values:
+#       - text:true
+#       - text:false
+# ...
```

Comment:
That's a neat idea. A couple questions:

(1) default value of this input? am thinking it should be true, and probably an advanced input, since many images ship with repo metadata stripped. (However, we won't waste your time with an update if RS_PACKAGES is empty.)

(2) Do we update when there are packages to install even if they have already been installed? This could add 30-60 seconds to boot time on reboot, because `apt-get update` is pretty heavyweight.

(3) Because of #2, it seems to me like if we want to run the pkgman update, we should also leave in the fancy check of which packages are already installed -- and only run update if we need to install something.